### PR TITLE
Feature/cg 1061

### PR DESCRIPTION
**Pull request by m-pizarro**
## Issue tracker links

_Add links to any relevant tasks/stories/bugs/pagerduty/etc_

*Example - dummy TODO project*

[TODO-123](https://autoclouddev.atlassian.net/browse/TODO-123)

## Changes/solution

_How does this change address the problem?_

## Testing

_Describe how the testing was done, plus evidence, if not covered by automated tests_

## Notes and considerations

_Add any additional notes and/or considerations_

## Dependencies

_Add dependencies on any other PRs, if applicable

### CODE REVIEW

File: src/services/ec2/connections.ts [hunk 1]
```diff
@@ -189,17 +193,37 @@ export default ({
...
     }
   }
 
   /**
-   * Find EKS
+   * Find EKS cluster
    * related to this EC2
    */
-  // TODO: Implement when eks service is ready
+  const eksClusterName = getEksClusterName(tags)
```
Comment (tyler-dunkel) [inline src/services/ec2/connections.ts:206]: Is the only way to get this connection through the tags on the ec2?

File: src/services/ec2/schema.graphql [hunk 2]
```diff
@@ -116,8 +116,10 @@ type awsEc2 @key(fields: "arn") {
   eip: [awsEip] @hasInverse(field: ec2Instance)
   networkInterfaces: [awsNetworkInterface] @hasInverse(field: ec2Instance)
```
Comment (tyler-dunkel) [inline src/services/ec2/schema.graphql:117]: for EC2 we need to make a connection to iamRole using the iamInstanceProfile property.

Comment (m-pizarro) [reply src/services/ec2/schema.graphql:112]: oh yes, I had missed it. Done!

**Comment by m-pizarro** [comment]
> 

ok, understand. I thinks It's not difficult. Let me check it out and anything I let you know.

**Review by tyler-dunkel — APPROVED** [review]

**Pull request merged**